app/styles/components/_rui-avatar.sass

```diff
@@ -1,14 +1,28 @@
 .rui-avatar
-  width: 2.2857142857rem // ~32px at base 14
-  height: auto
+  background: url($default-avatar-small) top left no-repeat
+  background-size: 100% 100%
+  border-radius: $border-radius
+  display: inline-block
+  overflow: hidden
+  vertical-align: middle
+  &,
```

Comment:
In this case, we are sizing the image and the enclosing span so when the image is loading/hidden, it has the same dims and placeholder image is painted.